pkg/kubelet/kubelet_pods.go

```diff
@@ -148,17 +148,17 @@ func (kl *Kubelet) makeBlockVolumes(pod *v1.Pod, container *v1.Container, podVol
 // Kubernetes will not mount /etc/hosts if:
 // - when the Pod sandbox is being created, its IP is still unknown. Hence, PodIP will not have been set.
 // - Windows pod contains a hostProcess container
-func shouldMountHostsFile(pod *v1.Pod, podIPs []string, supportsSingleFileMapping bool) bool {
```

Comment:
Update the comments? - Removing supportsSingleFileMapping removes the formal containerd constraint from this.